dlang.org.ddoc

```diff
@@ -27,10 +27,10 @@ _=
 
 COMMON_SCRIPTS =
     $(SCRIPTLOAD https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js)
-    $(SCRIPT window.jQuery || document.write('<script src="$(ROOT)/js/jquery-1.7.2.min.js">\x3C/script>'))
-    $(SCRIPTLOAD $(ROOT)/js/codemirror-compressed.js)
-    $(SCRIPTLOAD $(ROOT)/js/run.js)
-    $(SCRIPTLOAD $(ROOT)/js/cssmenu.js)
+    $(SCRIPT window.jQuery || document.write('<script src="$(ROOT_DIR)js/jquery-1.7.2.min.js">\x3C/script>'))
```

Comment:
The ROOT macro is still there if you need it without the slash.

I need this for CHM generation. We can either do it this way, or with more regular expressions. You decide.